packages/plugin-npm-cli/sources/commands/npm/publish.ts

```diff
@@ -5,6 +5,8 @@ import {npmConfigUtils, npmHttpUtils, npmPublishUtils}
 import {packUtils}                                                                              from '@yarnpkg/plugin-pack';
 import {Command, Option, Usage, UsageError}                                                     from 'clipanion';
 
+const {env} = process;
```

Comment:
Avoid making an indirection for process.env, it makes it a little more difficult to understand where a value comes from at a glance.